tests/sandbox/test_jax.py

```diff
@@ -23,6 +24,7 @@ def compare_jax_and_py(fgraph, inputs, cmp_fn=np.allclose):
     jax_res = theano_jax_fn(*inputs)
 
     if isinstance(jax_res, list):
+        # For some reason this returns [[array0, array1], None]
         assert all(isinstance(res, jax.interpreters.xla.DeviceArray) for res in jax_res)
```

Comment:
Here is the current issue, not sure why this is happening.

Reply:
The SVD `Op` is one of the rare ones that has two outputs.  That's why you're seeing this.